features/zoeken/met-burgerservicenummer/overzicht.feature

```diff
@@ -0,0 +1,46 @@
+# language: nl
+Functionaliteit: gegevens raadplegen van een persoon
+
+  Als consumer
+  wil ik de actuele gegevens van één of meer personen kunnen raadplegen door het opgeven van zijn/hun burgerservicenummer
+
+    Achtergrond:
+      Gegeven de persoon 'Robin' met burgerservicenummer '000000024'
+      En de persoon 'Saskia' met burgerservicenummer '000000025'
+      En de persoon 'Gerda' met burgerservicenummer '000000026'
+
+  Regel: een persoon moet worden gezocht met zijn burgerservicenummer om zijn actuele gegevens te kunnen raadplegen
+
+    Scenario: het burgerservicenummer van één persoon is opgegeven
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met burgerservicenummer van 'Robin'
+      Dan wordt 'Robin' gevonden
+
+    Scenario: het burgerservicenummer van meer personen is opgegeven
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met burgerservicenummer van 'Robin, Saskia en Gerda'
+      Dan worden 'Robin, Saskia en Gerda' gevonden
+
+  Regel: de optionele 'gemeenteVanInschrijving' parameter kan worden gebruikt om de zoek criteria aan te scherpen
```

Comment:
is dit niet ook de plek om het nut van deze parameter toe te lichten? Een bsn is al uniek, dus aanscherpen is eigenlijk niet nodig. Gaat om het afdwingen door gemeentelijke afnemers dat specifieke gebruikers alleen eigen ingezetenen kunnen raadplegen

Reply:
staat in de devportal, hoeft niet in de feature te worden uitgelegd waarom deze parameter bestaat.

Regel tekst wijzigen: "Om uitsluitend binnengemeentelijk te kunnen zoeken ..."

Reply:
done. Dit is ook aangepast in de andere zoek overzicht.feature files